### PR TITLE
Profile fix

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

File: resources/views/backend/users/show.blade.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
                                         <hr>
                                         </div>
                                         <div class="avata text-center">
-                                            <img src="{!! asset(config('app.img').$user->avatar) !!}" class="avata-img img-circle" alt="">
+                                            <img src="{!! asset(config('app.link_avatar').$user->avatar) !!}" class="avata-img img-circle" alt="">
```
Comment (BunnyPi04) [inline resources/views/backend/users/show.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Comment by BunnyPi04** [comment]
commented
conflict nhé

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Requests/ProfileUpdateFormRequest.php [hunk 2]
```diff
@@ -24,7 +24,8 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'avatar'=>'image',
```
Comment (BunnyPi04) [inline app/Http/Requests/ProfileUpdateFormRequest.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**